src/utils/createRequestHeaders.js

```diff
@@ -1,5 +1,9 @@
-export default async ({ getAccessToken, version } = {}) => ({
-  'Content-Type': 'application/json',
-  ...(getAccessToken && { Authorization: `Bearer ${await getAccessToken()}` }),
-  ...(version && { 'X-Api-Version': version }),
-});
+export default async ({ getAccessToken, version } = {}) => {
+  let accessToken = undefined;
+  return {
+    'Content-Type': 'application/json',
+    ...(getAccessToken &&
```

Comment:
Would probably be a bit more readable if we create a helper function for this `parseAccessToken`?

Reply:
But what would the `parseAccessToken` do? Or do you mean something more like `getAuthorizationHeader`?

And then do something like `...(getAccessToken && getAuthorizationHeader(getAccessToken))`